### PR TITLE
fix: fix snuba-outcomes-billing-consumer args

### PR DESCRIPTION
**Pull request by kimxogus**
`--raw-events-topic` arg was missing.

https://github.com/getsentry/self-hosted/pull/2909/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R270

### CODE REVIEW

**Comment by kimxogus** [comment]
@Mokto Can you review this?

File: sentry/templates/deployment-snuba-outcomes-billing-consumer.yaml [hunk 1]
```diff
@@ -80,6 +80,8 @@ spec:
           - "--consumer-group"
           - "snuba-consumers"
           - "--auto-offset-reset"
+          - "--raw-events-topic"
+          - "outcomes-billing"
           - "{{ .Values.snuba.outcomesBillingConsumer.autoOffsetReset }}"
```
Comment (farodin91) [inline sentry/templates/deployment-snuba-outcomes-billing-consumer.yaml:85]: This should be behind auto offset reset

**Comment by rdziwinski** [comment]
Now there is another problem:
`Error: Invalid value for '--auto-offset-reset': '--raw-events-topic' is not one of 'error', 'earliest', 'latest'.`
Is it should be like below?

`--auto-offset-reset=earliest --raw-events-topic outcomes-billing`

**Comment by farodin91** [comment]
https://github.com/sentry-kubernetes/charts/pull/1257

**Pull request merged**